### DISCUSSION TITLE
Is it legal to use github pages to load resources for another site

### DISCUSSION THREAD

**Discussion by lightning-speed** [community/community#39897] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So I am making a game and to load some resources and I am using github pages to fetch those res for my game site that is hosted by another service provider
so that I can save money on my service provider
is it fine?

**Comment by radar-blip** [comment]
Should be fine, yes. Long as those resources aren't to large in file size.

**Reply by lightning-speed** [reply]
ok, thanks for the help :tada:

**Reply by radar-blip** [reply]
No problem. Good luck!

**Reply by radar-blip** [reply]
Here's a source that has some info on `prohibited uses` (a little past halfway down the page).
https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages
You should be alright.
